### ISSUE TITLE
[4.1] !important rule gets dropped from `animation` if `removeUnusedAtRules` is enabled

### ISSUE THREAD

**Issue by ngyikp** [jakubpawlowicz/clean-css#938]
### Environment

* clean-css version - `npm ls clean-css`: 4.1.0
* node.js version - `node -v`: v7.10.0
* operating system: macOS Sierra 10.12.4

### Configuration options

```js
const fs = require('fs');
const cleanCss = require('clean-css');

const compressor = new cleanCss({
	level: {
		2: {
			removeUnusedAtRules: true
		}
	}
});

const minified = compressor.minify(fs.readFileSync('test.css', 'utf-8'));
console.log(minified.styles);
```

### Input CSS

```css
@-webkit-keyframes abc {
	from {
		background-position: 0 0;
	}
	to {
		background-position: 10px 10px;
	}
}
@keyframes abc {
	from {
		background-position: 0 0;
	}
	to {
		background-position: 10px 10px;
	}
}
.a {
	-webkit-animation: none !important;
	animation: none !important;
}
.c {
	-webkit-animation: 4s ease-in-out infinite abc !important;
	animation: 4s ease-in-out infinite abc !important;
}
```

### Actual output CSS

(prettied for easier reading)

```css
@-webkit-keyframes abc {
	from { background-position: 0 0 }
	to { background-position: 10px 10px }
}
@keyframes abc {
	from { background-position: 0 0 }
	to { background-position: 10px 10px }
}
.a { -webkit-animation: none; animation: none }
.c { -webkit-animation: 4s ease-in-out infinite abc; animation: 4s ease-in-out infinite abc }
```

### Expected output CSS

The `!important` rule gets dropped from `animation` if `removeUnusedAtRules` is enabled.

**Comment by jakubpawlowicz** [comment]
Ooops! Thanks @ngyikp for reporting this so quickly. Fix is under way!

**Comment by jakubpawlowicz** [comment]
Fixed in 4.1.1, thanks @ngyikp!
